### PR TITLE
Spring-02

### PR DESCRIPTION
**Pull request by pavlokhomiak**

### CODE REVIEW

**Review by Rudiachenko — APPROVED** [review]
good job

File: src/main/java/config/AppConfig.java [hunk 1]
```diff
@@ -4,25 +4,31 @@
...
 
 @Configuration
+@PropertySource("classpath:db.properties")
 @ComponentScan(basePackages = {
         "service.impl",
         "dao.impl"
 })
 public class AppConfig {
+    @Autowired
+    private Environment env;
```
Comment (Sofasmile) [inline src/main/java/config/AppConfig.java]: What is better: setter, constructor, or field injection?

Comment (Sofasmile) [inline src/main/java/config/AppConfig.java]:
```suggestion
    private Environment environment;
```

File: src/main/java/config/AppConfig.java [hunk 2]
```diff
@@ -32,8 +38,8 @@ public LocalSessionFactoryBean getSessionFactory() {
         localSessionFactoryBean.setDataSource(getDataSource());
 
         Properties prop = new Properties();
```
Comment (Sofasmile) [inline src/main/java/config/AppConfig.java]:
```suggestion
        Properties properties = new Properties();
```

File: src/main/java/config/AppConfig.java [hunk 3]
```diff
@@ -19,16 +18,20 @@
         "dao.impl"
 })
 public class AppConfig {
-    @Autowired
-    private Environment env;
+
+    private Environment environment;
```
Comment (Sofasmile) [inline src/main/java/config/AppConfig.java:22]:
```suggestion
    private final Environment environment;
```

**Review by Sofasmile — APPROVED** [review]

**Pull request merged**